fastapi/app/main.py

```diff
@@ -0,0 +1,231 @@
+"""Module providing SpiceQL endpoints"""
+
+from ast import literal_eval
+from typing import Annotated, Any, Optional
+from fastapi import FastAPI, Query
+from pydantic import BaseModel, Field
+import pyspiceql
+
+
+# Models
+class MessageItem(BaseModel):
+    message: str
+
+class BodyModel(BaseModel):
+    result: Optional[Any] = None
+    error: Optional[str] = None
+
+class ResultModel(BodyModel):
+    result: Any = Field(serialization_alias='return')
+
+class ErrorModel(BodyModel):
+    error: str
+
+class ResponseModel(BaseModel):
+    statusCode: int
+    body: dict | str
+
+# Create FastAPI instance
+app = FastAPI()
+
+# General endpoints
+@app.get("/")
+async def root():
+    return {"message": "Visit the /docs endpoint to see the Swagger UI."}
```

Comment:
So this response does not render through the ResponseModel. Is that intentional? If so, why?

Reply:
This is intentional, it's currently a placeholder for when you hit the 127.0.0.1:8080/ root endpoint. I'm open to other ideas for the root endpoint!

The `ResponseModel` is structured the way it is to mimic the current SpiceQL's Lambda returns, which is:
```
{"statusCode": 200, "body": {"return": <insert-result>}
```
or 
```
{"statusCode": 500, "error": <error-message>
```
Feel free to ping me and I can point you to where the lambda code is located for reference.

Reply:
Ahh gotcha. Makes sense. I might consider a 302 with a redirect to /docs if someone hits the root, but this is just as cool.